### PR TITLE
identity governance: add examples of adding resources and resource roles in entitlement management

### PR DESCRIPTION
**Pull request by markwahl-msft**

### CODE REVIEW

File: api-reference/beta/api/accesspackageassignmentrequest-list.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ GET /identityGovernance/entitlementManagement/accessPackageAssignmentRequests
 
 ## Optional query parameters
 
-This method supports some of the OData query parameters to help customize the response. For example, to retrieve the access package of each request, include `$expand=accessPackage` in the query.  To retrieve only requests for a specific access package, include in the query a filter such as `$expand=accessPackage&$filter=accessPackage/id eq '9bbe5f7d-f1e7-4eb1-a586-38cdf6f8b1ea'`.  To retrieve the resulting assignment, include `$expand=accessPackageAssignment` in the query.
+This method supports some of the OData query parameters to help customize the response. For example, to retrieve the access package of each request, include `$expand=accessPackage` in the query.  To retrieve only requests for a specific access package, include in the query a filter such as `$expand=accessPackage&$filter=accessPackage/id eq '9bbe5f7d-f1e7-4eb1-a586-38cdf6f8b1ea'`.  To retrieve the resulting assignment, include `$expand=accessPackageAssignment` in the query.  To obtain more details on the requestor, include `$expand=requestor` in the query.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackageassignmentrequest-list.md:38]:
```suggestion
This method supports the `$expand` and `$filter` OData query parameters to help customize the response. For example, to retrieve the access package of each request, include `$expand=accessPackage` in the query.  To retrieve only requests for a specific access package, include in the query a filter such as `$expand=accessPackage&$filter=accessPackage/id eq '9bbe5f7d-f1e7-4eb1-a586-38cdf6f8b1ea'`.  To retrieve the resulting assignment, include `$expand=accessPackageAssignment` in the query.  To obtain more details on the requestor, include `$expand=requestor` in the query.
```

File: api-reference/beta/api/accesspackagecatalog-list-accesspackageresourceroles.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Retrieve a list of [accessPackageResourceRole](../resources/accesspackageresourcerole.md) objects of an [accessPackageResource](../resources/accesspackageresource.md) in an [accessPackageCatalog](../resources/accesspackagecatalog.md).  This list of roles can then be used by the caller to select a role, which is needed when subsequently [creating an accessPackageResourceRoleScope](accesspackage-post-accesspackageresourcerolescopes.md).
+Retrieve a list of [accessPackageResourceRole](../resources/accesspackageresourcerole.md) objects of an [accessPackageResource](../resources/accesspackageresource.md) in an [accessPackageCatalog](../resources/accesspackagecatalog.md). The resource would be one which was previously added to the catalog by [creating an accessPackageResourceRequest](accesspackageresourcerequest-post.md). This list of roles can then be used by the caller to select a role, which is needed when subsequently [creating an accessPackageResourceRoleScope](accesspackage-post-accesspackageresourcerolescopes.md).
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackagecatalog-list-accesspackageresourceroles.md:16]:
```suggestion
Retrieve a list of [accessPackageResourceRole](../resources/accesspackageresourcerole.md) objects of an [accessPackageResource](../resources/accesspackageresource.md) in an [accessPackageCatalog](../resources/accesspackagecatalog.md). The resource should have been added to the catalog by [creating an accessPackageResourceRequest](accesspackageresourcerequest-post.md). This list of roles can then be used by the caller to select a role, which is needed when subsequently [creating an accessPackageResourceRoleScope](accesspackage-post-accesspackageresourcerolescopes.md).
```

File: api-reference/beta/api/accesspackagecatalog-list-accesspackageresourceroles.md [hunk 3]
```diff
@@ -115,6 +117,61 @@ Content-type: application/json
 }
 ```
 
+### Example 2: Retrieving the roles of a resource for a SharePoint Online site
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackagecatalog-list-accesspackageresourceroles.md:120]:
```suggestion
### Example 2: Retrieve the roles of a resource for a SharePoint Online site
```

File: api-reference/beta/api/accesspackagecatalog-list-accesspackageresourceroles.md [hunk 4]
```diff
@@ -115,6 +117,61 @@ Content-type: application/json
...
+
+#### Request
+
+The following is an example of the request.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackagecatalog-list-accesspackageresourceroles.md:124]: Proposing that you explain this request further. I think it'll help give context to what you described in the description. Example: https://docs.microsoft.com/en-us/graph/api/accessreviewscheduledefinition-post?view=graph-rest-beta&tabs=http#example-2-create-an-access-review-on-all-teams-with-inactive-guest-users

Same for the response.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/accesspackagecatalog-list-accesspackageresourceroles.md (hunk 4 above)
Comment (markwahl-msft) [reply api-reference/beta/api/accesspackagecatalog-list-accesspackageresourceroles.md]: Added more text to this example, thanks.

**Pull request merged**